tensorflow/models/image/cifar10/cifar10_eval.py

```diff
@@ -81,36 +81,36 @@ def eval_once(saver, summary_writer, top_k_op, summary_op):
       print('No checkpoint file found')
       return
 
-    # Start the queue runners.
```

Comment:
Sorry, my bad. I had it dedented in my fork and then accidentally indented one level too much.